src/taipy/core/_repository/_sql_repository.py

```diff
@@ -58,7 +59,9 @@ def __init__(
 
         try:
             # More sql databases can be easily added in the future
-            self.engine = create_engine(f"sqlite:///{properties.get('db_location')}")
+            self.engine = create_engine(
+                f"sqlite:///{properties.get('db_location')}?check_same_thread=False", poolclass=StaticPool
```

Comment:
What are these arguments for?

Reply:
By default, when SQLite is created in one thread, it can not be use elsewhere.

The `poolclass=StaticPool` makes sure that there is only one connection among all threads.
And `?check_same_thread=False` allows the connection to be used by any thread.

Reply:
OK thanks.